### PR TITLE
feat(j-s): Add a isStatementDeadlineExpired field to our API

### PR DESCRIPTION
**Pull request by oddsson**
# Add a isStatementDeadlineExpired field to our API

[Asana](https://app.asana.com/0/1199153462262248/1204298511753938/f)

## What

Add a new field, `isStatementDeadlineExpired`, to our API. This deadline represents whether or not the deadline to hand in the statement document in appealed cases (frestur til að skila inn greinargerð) has passed. This deadline is 24 hours after a case is appealed.

## Checklist:

- [x] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] Formatting passes locally with my changes
- [x] I have rebased against main before asking for a review

### CODE REVIEW

File: apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts [hunk 1]
```diff
@@ -176,4 +178,55 @@ describe('transformCase', () => {
...
+
+      // Act
+      const res = transformCase(theCase)
+
+      // Assert
+      expect(res.isStatementDeadlineExpired).toBe(false)
+    })
+
+    it('should be false when more than one day has passed since the case was appealed', () => {
+      // Arrange
```
Comment (unakb) [inline apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts:195]: Shouldn't it be false "when lass than one day has passed"? and then in line 214 "when more than one day"

**Review by unakb — CHANGES_REQUESTED** [review]
I just have a concern because I'm not sure if I'm misunderstanding the statement deadline or if the logic here might be wrong. I thought that the statement deadline started ticking down when the judge has marked the case as received? Not when the case is appealed

**Review by unakb — APPROVED** [review]
There's one linting warning in the transformer and then the failing tests, but otherwise looks good

File: apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts (hunk 1 above)
Comment (oddsson) [reply apps/judicial-system/api/src/app/modules/case/interceptors/case.transformer.spec.ts:195]: Yes! Thanks

**Pull request merged**